draft-irtf-cfrg-vdaf.md

```diff
@@ -1371,31 +1375,34 @@ cross protocol attacks; see {{deep}}.
 
 ## Preparation {#sec-vdaf-prepare}
 
-~~~~
+~~~~ aasvg
     Aggregator 0   Aggregator 1           Aggregator SHARES-1
     ============   ============           ===================
 
     input_share_0  input_share_1          input_share_[SHARES-1]
       |              |                 ...  |
-      V              V                      V
+      v              v                      v
     +-----------+  +-----------+          +-----------+
     | prep_init |  | prep_init |          | prep_init |
-    +-----------+  +-----------+          +-----------+
+    +-+-------+-+  +-+-------+-+          +-+-------+-+
       |       |      |       |         ...  |       |
-      V       |      V       |              V       |
-    +---------|--------------|----------------------|-+   \
-    |         |              | prep_shares_to_prep  | |   |
-    +---------|--------------|----------------------|-+   |
+      v       |      v       |              v       |
+    +---------|--------------|----------------------|--+  \
+    |         |              |  prep_shares_to_prep |  |  |
+    +-+-------|------+-------|--------------+-------|--+  |
       |       |      |       |         ...  |       |     |
-      V       V      V       V              V       V     | x ROUNDS
+      v       v      v       v              v       v     | x ROUNDS
     +-----------+  +-----------+          +-----------+   |
     | prep_next |  | prep_next |          | prep_next |   |
-    +-----------+  +-----------+          +-----------+   |
+    +-+-------+-+  +-+-------+-+          +-+-------+-+   |
       |       |      |       |         ...  |       |     |
-      V       V      V       V              V       V     /
+      v       v      v       v              v       v     /
```

Comment:
We could make this a curved line as follows:

```
    +---------|--------------|----------------------|--+  -.
    |         |              |  prep_shares_to_prep |  |    |
    +-+-------|------+-------|--------------+-------|--+    |
      |       |      |       |         ...  |       |       |
      v       v      v       v              v       v       | x ROUNDS
    +-----------+  +-----------+          +-----------+     |
    | prep_next |  | prep_next |          | prep_next |     |
    +-+-------+-+  +-+-------+-+          +-+-------+-+     |
      |       |      |       |         ...  |       |       |
      v       v      v       v              v       v     -'
```

Reply:
nit: I changed the number of columns between the curved line and the rest of the figure from 2 to 1, in order to accommodate the artwork width requirements.